utils/wrap_with_fhir_tuple_test.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020, Tranquil Data, Inc. All rights reserved.
+ */
+
+package utils
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+)
+
+func TestThatStructsCanBeParsed(t *testing.T) {
+	st, err := ParseFhirModel("../models/activitydefinition.go")
+	if err != nil {
+		t.Error(err)
+	}
+	if st == nil {
+		t.Errorf("Struct was nil")
+	}
+	emitted, err := EmitTupleCode(st, "models")
+	if err != nil {
+		t.Error(err)
+	}
+	if len(emitted) == 0 {
+		t.Errorf("emitted was empty")
+	}
+	err = ioutil.WriteFile("../models/tranquil_activitydefinition.go", []byte(emitted), 0644)
```

Comment:
Are there any concerns about permissions issues that might prevent writing files in unit tests?  Should these be written to a tmp directory instead?

Reply:
actually this write shouldn't be happening at all. Remnant of a bygone age.